atcoder/abc138C_alchemist.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
```

Comment:
add a comment with the following metadata:
- author
- link to the problem
- category

something like this: 
https://github.com/Vicfred/kyopro/blob/2a4f7b7e49251ded7985da5a2bf3a1a964a1d8cb/codeforces/cf1312C_adding_powers.cpp